wp-content/themes/access/views/programs/single.twig

```diff
@@ -30,7 +30,7 @@
       </div>
 
       <div>
-        <h1 class="c-header__descriptor" itemprop="alternateName">
+        <h1 class="c-header__descriptor" itemprop="alterName">
```

Comment:
Is this right? `alterName` instead of `alternateName`?

Reply:
@devowhippit I think for this one is `alternameName` https://schema.org/alternateName

Reply:
Ok. The change looks misspelled so make sure it's correct. Thanks!